docs/existingLicense.md

```diff
@@ -15,9 +15,10 @@ Use either of these two methods to make an existing product license file availab
 * Copy each license file to the server profile location associated with the product. The default server profile locations are:
   - PingFederate: `instance/server/default/conf/pingfederate.lic`
   - PingAccess: `instance/conf/pingaccess.lic`
-  - PingDirectory: `instance/pingdirectory.lic`
-  - PingDataGovernance: `instance/pingdatagovernance.lic`
-  - PingDataSync: `instance/pingdatasync.lic`
+  - PingDirectory: `instance/PingDirectory.lic`
+  - PingDataGovernance: `instance/PingDataGovernance.lic`
+  - PingDataGovernance PAP: `instance/PingDataGovernance.lic`
+  - PingDataSync: `instance/PingDirectory.lic`
```

Comment:
This looks strange, but it's consistent with the product, and was also correct in the "Expected license file name" table below.

Reply:
thanks for fixing!